hack/os-packages/Makefile

```diff
@@ -1,31 +1,37 @@
 RPM_PACKAGES_GETTER_IMAGE = mesosphere/rpm-artifacts-getter
 KUBERNETES_RPM_PATCH = 0
+rhel_major_version ?= 7
+fips ?= 0
 
 .PHONY: os-packages-artifacts
-os-packages-artifacts: download-all-rpms artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_fips_rpms.tar.gz
+os-packages-artifacts: archive-os-packages.$(fips)
 
-artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz:
-	cd artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el7_x86_64_rpms && tar -czf ../os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_rpms.tar.gz *
+# fips disabled
+.PHONY: archive-os-packages.0
+archive-os-packages.0: artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el$(rhel_major_version)_x86_64_rpms.tar.gz
 
-artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_fips_rpms.tar.gz:
-	cd artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el7_x86_64_fips_rpms && tar -czf ../os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_x86_64_fips_rpms.tar.gz *
+# fips enabled
+,PHONY: archive-os-packages.1
+archive-os-packages.1: artifacts/os-packages_$(DEFAULT_KUBERNETES_VERSION_SEMVER)_el$(rhel_major_version)_x86_64_fips_rpms.tar.gz
```

Comment:
Does adding `fips` in file name OK? ideally, I think artifacts directory should have similar layout as release S3 bucket for the rpms. That way we can just use `s3_sync` from our automation scripts.

Any suggestion for naming the file? 
Should we add directories ?
thinking `artifacts\os-packages\<OS>-<VERSION>\<arch>\<fips|PWD>/packages_<k8s_version>.tar.gz`

OR similar to like OS mirrors are set up https://mirrors.edge.kernel.org/centos/7/extras/x86_64/Packages/
`<OS>/<MAJOR_VERSION>/<K8S_VERSION>/<ARCH>/<FIPS|NON-FIPS?>/<packages|pip-packages|images>/packages_<k8s_version>.tar.gz`

Reply:
I prefer `artifacts/os-packages/<ansible_distribution>/<ansible_distribution_major_version>/<arch>/os-packages_{{ kubernetes_version }}_<arch>_<fips|"">.tar.gz` because when pushed to S3 all fips packages will end up in a subdirectory.